src/contents/en-gb/home.yaml

```diff
@@ -46,6 +46,18 @@ contribute:
     link:
       name: Core Technology
       href: /core_technology
+sharing:
+  title:
+    shared: Stand alone
```

Comment:
The alignment of 'Stand alone technology' title is too low (at full width screen on laptop

Reply:
adjusted (was the same diff as section above but looked bigger as section is smaller)